ddtrace/propagation/_database_monitoring.py

```diff
@@ -86,6 +106,14 @@ def _get_dbm_comment(self, db_span):
             DBM_DATABASE_SERVICE_NAME_KEY: service_name_key,
         }
 
+        db_instance = self._find_first_tag(db_span, DBM_PEER_DB_INSTANCE_SOURCE_TAGS)
```

Comment:
Are we able to choose to ignore this if peer service isn't enabled?  Its a minor optimization if we can, so fine to merge either way.